### PR TITLE
[Issue #649 fix] Fixed the erratic behavior of Calculator

### PR DESCRIPTION
**Pull request by dhruvmisra**
The issue has been fixed. I have also made the history scrolling independent of the calculator, providing perhaps a better user experience.

after

### CODE REVIEW

**Comment by s-mag** [comment]
@dhruvmisra This will not work when the height is less... You can do one thing- Just make the toolbar responsive which will solve the issue...

File: activities/Stopwatch.activity/css/activity.css [hunk 1]
```diff
@@ -144,4 +144,4 @@ button.remove {
 
 .buttons-group button.clear-marks-button {
   background-image: url(../icons/edit-clear.svg);
-}
```
Comment (s-mag) [inline activities/Stopwatch.activity/css/activity.css]: No need to make a change where not required.

File: activities/Stopwatch.activity/js/activity.js [hunk 2]
```diff
@@ -263,4 +263,4 @@ define(["sugar-web/activity/activity","mustache", "sugar-web/env"], function (ac
 
     });
 
-});
```
Comment (s-mag) [inline activities/Stopwatch.activity/js/activity.js]: No need to make a change where not required.

**Comment by dhruvmisra** [comment]
@s-mag Thank you for the review. The actual issue I was tackling here was the irregular resizing of the calculator container (Issue #649). I have created a PR which makes the toolbar responsive for Home views. (PR #647) Kindly review the same and I shall implement it for other activities too.

**Comment by s-mag** [comment]
Sugarizer activities are not meant or designed to work in small screens. See #422
There is no need to correct all these activities as no one will use them in such a small screen size.These activities work perfectly in normal conditions...

File: activities/Stopwatch.activity/css/activity.css (hunk 1 above)
Comment (dhruvmisra) [reply activities/Stopwatch.activity/css/activity.css]: There was only a removal of new line character. Fixed it.

File: activities/Stopwatch.activity/js/activity.js (hunk 2 above)
Comment (dhruvmisra) [reply activities/Stopwatch.activity/js/activity.js]: There was only a removal of new line character. Fixed it.

**Comment by llaske** [comment]
Not an issue.

**Pull request closed without merging**